muxers/mplex/src/codec.rs

```diff
@@ -244,25 +244,25 @@ impl Encoder for Codec {
     fn encode(&mut self, item: Self::Item, dst: &mut BytesMut) -> Result<(), Self::Error> {
         let (header, data) = match item {
             Frame::Open { stream_id } => {
-                (u64::from(stream_id.num) << 3, Bytes::new())
+                (stream_id.num << 3, Bytes::new())
```

Comment:
Thanks! I should have caught this in https://github.com/libp2p/rust-libp2p/pull/2094/.